test/client/spec/mock-components.js

```diff
@@ -0,0 +1,87 @@
+import React, { PropTypes } from "react";
+import { Data } from "src/index";
+import { defaults, get, reduce, map } from "lodash";
+
+class MockDataComponent extends React.Component {
+  static displayName = "MockDataComponent";
+  static role = "dataComponent";
+
+  static propTypes = {
+    datum: PropTypes.object,
+    events: PropTypes.object,
+    style: PropTypes.object
+  };
+
+  render() {
+    const { datum: { x, y }, events, style } = this.props;
+    return (
+      <p style={style} {...events}>
+        `${x}: ${y}`
+      </p>
+    );
+  }
+}
+
+class MockLabel extends React.Component {
+  static displayName = "MockLabel";
+  static role = "label";
+
+  static propTypes = {
+    text: PropTypes.string
+  };
+
+  render() {
+    const { text } = this.props;
+
+    return (
+      <p>`${text}`</p>
+    );
+  }
+}
+
+class MockChart extends React.Component {
```

Comment:
*nitpick* this should be renamed as it does not mock the functionality of VictoryChart. Maybe "MockVictoryComponent"?